raiden-ts/src/raiden.ts

```diff
@@ -1256,6 +1265,25 @@ export class Raiden {
     );
     return receipt.transactionHash as Hash;
   }
+
+  public async planUdcWithdraw(value: BigNumberish): Promise<Hash> {
+    const withdrawPlan = merge(
+      this.action$.pipe(
+        filter(udcWithdraw.failure.is),
```

Comment:
If you have `.success` action as the role currently occupied by `Planned`, you can use `asyncActionToPromise` utility to wait for the success/failure response. It may be important to have a meaningful `meta` for it, but would work with empty object as well.